config/clock.rb

```diff
@@ -21,6 +21,8 @@ class Clock
   every(1.hour, 'SendChaseEmailToCandidates', at: '**:40') { SendChaseEmailToCandidatesWorker.perform_async }
   every(1.hour, 'SendCourseFullNotifications', at: '**:45') { SendCourseFullNotificationsWorker.perform_async }
 
+  every(1.hour, 'RejectAwaitingReferencesCourseChoices', at: '**:50') { RejectAwaitingReferencesCourseChoicesWorker.perform_async }
```

Comment:
This should really only need to run once. Maybe every(1.month, .....

Reply:
Could it miss the window when it's meant to run if we run it only once a month?